recipes/libe57format/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class LibE57FormatConan(ConanFile):
+    name = "libe57format"
+    license = "BSL-1.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/asmaloney/libE57Format"
+    description = "Library for reading & writing the E57 file format"
+    topics = ("e57", "io", "point-cloud")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "cmake", "cmake_find_package"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("xerces-c/3.2.3")
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, "11")
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["BUILD_SHARED_LIBS"] = self.options.shared
+        if self.settings.os == "Windows":
+            self._cmake.definitions["USING_STATIC_XERCES"] = True
+        self._cmake.configure(build_folder=self._build_subfolder,
+                              source_folder=self._source_subfolder)
+        return self._cmake
+
+    def build(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("LICENSE.md", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+
+    def package_info(self):
+        self.cpp_info.libs = ["E57Format"]
+        self.cpp_info.names["cmake_find_package"] = "E57Format"
+        self.cpp_info.names["cmake_find_package_multi"] = "E57Format"
+        self.cpp_info.names["pkg_config"] = "E57Format"
```

Comment:
```suggestion
        self.cpp_info.filenames["cmake_find_package"] = "e57format"
        self.cpp_info.filenames["cmake_find_package_multi"] = "e57format"
        self.cpp_info.names["pkg_config"] = "e57format"
        if self.settings.os in ["Linux", "FreeBSD"]:
            self.cpp_info.system_libs.append("pthread")
```

Reply:
I don't find any mention of a `.pc` file in the [cmake script](https://github.com/asmaloney/libE57Format/blob/b5b76616836754012cc8e7809bd19cee00539d3c/CMakeLists.txt#L132-L154).
So I'ld remove `self.cpp_info.names["pkg_config"] = "e57format"`

Reply:
Agreed